sympy/physics/vector/vector.py

```diff
@@ -345,12 +339,20 @@ def __ror__(self, other):
     def __rsub__(self, other):
         return (-1 * self) + other
 
-    def __str__(self, printer=None):
+    def __str__(self, printer=None, order=True):
```

Comment:
I made this change because the dictionary is unordered and each runs prints the same vector differently. Index was to make sure that the printing is the same across multiple runs. Let me know if there is a better way.